Assets/Scripts/DepositManager.cs

```diff
@@ -76,11 +78,13 @@ void Update()
 					mPhotonView.RPC("DepositCoin", PhotonTargets.All, viewId);
```

Comment:
(this comment should be on line 68 but I can't figure out how to get it marked for that line)

We are able to get the player associated with a viewid therefore we can know who the player in the zone is and therefore can get the number of coins that the player has.

Basically I think there should be  a relatively painless way of getting the players coins in the zone :D

something like...

`PhotonView.Find(viewId).gameObject.GetComponent<CoinInventory>().GetNumberOfCoins()`

or something like that